lib/membrane_overlay_filter.ex

```diff
@@ -0,0 +1,114 @@
+defmodule Membrane.OverlayFilter do
+  @moduledoc """
+  Applies image or text overlay to video.
+
+  Based on `Image`.
+  """
+  use Membrane.Filter
+
+  alias Membrane.RawVideo
+
+  def_input_pad :input, accepted_format: %RawVideo{pixel_format: :I420}
+  def_output_pad :output, accepted_format: %RawVideo{pixel_format: :I420}
+
+  alias Vix.Vips.Image, as: Vimage
+  alias Vix.Vips.Operation
+
+  def_options overlay: [
+                spec: String.t() | Vix.Vips.Image.t(),
```

Comment:
How about:
```suggestion
                spec: Path.t() | Vix.Vips.Image.t(),
```
?
I was slightly confused that perhaps we can provide the text which will be put on the image :D